frontend/micro-ui/web/micro-ui-internals/packages/modules/Measurement/src/pages/employee/ViewMeasurement.js

```diff
@@ -1,120 +1,60 @@
-import { Header, Card, Loader, WorkflowActions, CardSubHeader } from "@egovernments/digit-ui-react-components";
-import { transformEstimateData } from "../../utils/transformEstimateData";
+import { Header, Card, Loader, ViewComposer } from "@egovernments/digit-ui-react-components";
 import React, { useState, useEffect } from "react";
 import { useTranslation } from "react-i18next";
-import ApplicationDetails from "../../../../templates/ApplicationDetails";
-import MeasurementHistory from "../../components/MBHistoryTable";
-import MeasureTable from "../../components/MeasureTable";
+import { data } from "../../configs/ViewMeasurementConfig";
+import { getThumbnails } from "../../../../../libraries/src/utils/thumbnail";
```

Comment:
put this service inside filestore or any service hook and access through DIgit. keyword